core/domain/user_services.py

```diff
@@ -280,7 +446,21 @@ def get_profile_pictures_by_user_ids(user_ids):
 
 
 def get_user_settings(user_id, strict=False):
-    """Return the user settings for a single user."""
+    """Return the user settings for a single user.
+
+    Args:
+        user_id: str. The user id.
+        strict: bool. Whether to fail noisily if no user with the given
+            id exists in the datastore. Defaults to False.
+
+    Returns:
+        UserSettings or None. If the given user_id does not exist and strict is False,
+        returns None. Otherwise, returns the corresponding UserSettings domain object.
+
+    Raises:
+        Exception: If strict is True and given user_id does not exist.
```

Comment:
Drop "if":

Exception: `strict` is True, and the given user_id does not exist.